site/content/integrate/apps/deployment-aws/deploy-to-aws/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Deployment (Admins)"
+heading: "Deploying AWS Serverless Apps (For Admins)"
+description: "TODO"
+weight: 40
+---
+
+Admins of the on-prem Mattermost instance must prepare both AWS and Mattermost before provisioning and deploying an app.
+
+Note that apps are not supported in the Mattermost Cloud - they are only used for developer testing convenience.
+
+Provisioning in the AWS cloud environment will be done by the **appsctl** tool described in the AWS Setup section.
+
+We need an app bundle to provision an app. The bundle will be provisioned from S3 and consists of three parts which will be completed via the **appsctl** tool:
+
+1. Creating the lambda functions with appropriate policies.
+1. Storing static assets in the dedicated S3 bucket.
+1. Storing the app’s manifest file in the same dedicated S3 bucket.
+
+### Mattermost setup
+
+Follow the Prerequisites section of the [JavaScript]({{< ref "quick-start-js" >}}) or [Go]({{< ref "quick-start-go" >}}) Quick Start Guides to set up your Mattermost instance. Ensure that you install the [apps plugin]({{< ref "quick-start-js#install-the-apps-plugin" >}}) as part of this process.
+
+### AWS setup
+
+### 1. Create AWS S3 bucket
+
+You will need to create an S3 bucket within AWS or use an existing bucket.
+
+1. Visit https://s3.console.aws.amazon.com
+1. Create a bucket.
+1. **Bucket name**: Give your bucket a name.
+1. **AWS Region**:
+    1. Select your region
+    1. Save the slug value for later (Example: `us-east-1`)
+    1. Corresponding Lambdas will be provisioned in the same region
+1. **Block Public Access settings for this bucket**
+    1. Uncheck **Block *all* public access**
+    1. Check **Block public access to buckets and objects granted through new public bucket or access point policies**
+    1. Check **Block public and cross-account access to buckets and objects through any public bucket or access point policies**
+    1. Check **I acknowledge that the current settings might result in this bucket and the objects within becoming public.**
```

Comment:
See https://github.com/mattermost/mattermost-developer-documentation/pull/826#discussion_r648470561

Reply:
Fixed in 1407c306